privacy.md

```diff
@@ -0,0 +1,70 @@
+# Privacy policy
+
+## Preamble and scope
+
+This privacy policy governs the user data processing executed by LINAGORA, a simplified stock company (SAS) with a capital of 1 552 980 euros and a registered office located 37 rue Pierre Poli - CS 60238, 92441 Issy-les-Moulineaux Cedex (FRANCE), registered with the Registry of Trade and Companies of Nanterre (France) under SIRET number 431 473 669 00130, and/or any subsidiary of this company, hereinafter referred to as "LINAGORA".
+
+LINAGORA is the Data Processor for your personal data.
+
+LINAGORA services business model is not based on your user personal data. Our business model is based on offering support and other services complementary to our Services platform. WE DO NOT SELL YOUR DATA, AND YOU ARE NOT THE PRODUCT (AND NEITHER IS YOUR PERSONAL DATA).
+
+## Definitions
+
+The following expressions in the Privacy policy have the meaning indicated in this section:
+
+Content : Content is defined as any data, regardless of nature (whether information, image, video, audio, file, sign, text, signal, program, software, code, or any other element) which is available (displayed and/or downloadable) through the Services platform. User Content is a specific type of Content.
```

Comment:
can you more details on the data which we request to access to user storage: files, images, videos, contacts which we only use for the functionality of the app, user still decline it and still use the app? bla bla bla

Reply:
Then please give me the details for all permission:

Which permission do we require?

And why do we need such permission...

I am awaiting in front of the gate of the airport so if you want that quickly I suggest you push a fixup directly to my branch.

Reply:
It is Functionality and optional (user can reject it and still use the app)
- Photos and Videos
- Audio files
- Files and docs 
- Contacts